### DISCUSSION TITLE
Unable to recieve Incoming Mails

### DISCUSSION THREAD

**Discussion by engrpeters** [postalserver/postal#2316] [Help with using Postal]
This is an example error message from gmail 

The recipient server did not accept our requests to connect. Learn more at https://support.google.com/mail/answer/7720 [[postal.theneighborhood.com.ua](http://postal.theneighborhood.com.ua/). 2603:1020:201:f::fe: FAILED_PRECONDITION: connect error (111): Connection refused] [[postal.theneighborhood.com.ua](http://postal.theneighborhood.com.ua/). [20.107.50.171](http://20.107.50.171/): FAILED_PRECONDITION: connect error (111): Connection refused]

N:B I only have port 587 opened 

Another issue I'm able to send emails to gmail but not yahoo.

What do you suggest I do please ?

**Comment by willpower232** [comment]
> N:B I only have port 587 opened

this is probably related, Postal defaults to 25 which should be enough for usual use cases. You could use multiple ports if you wanted but this tends to confuse the situation more often than not.
